pom.xml

```diff
@@ -21,6 +22,40 @@
     </license>
   </licenses>
 
+  <developers>
+    <developer>
+      <name>Patrick Fink</name>
+      <email>mail@pfink.de</email>
+      <organization>openHAB</organization>
+      <organizationUrl>http://www.openHAB.org</organizationUrl>
+    </developer>
+    <developer>
+      <name>Kai Kreuzer</name>
+      <email>kai@openhab.org</email>
+      <organization>openHAB</organization>
+      <organizationUrl>http://www.openHAB.org</organizationUrl>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:git@github.com:openhab/infrastructure.git</connection>
+    <developerConnection>scm:git:git@github.com:openhab/infrastructure.git</developerConnection>
```

Comment:
Don't wee need `https://` as a url prefix here?

Reply:
where do you see a https?

Reply:
fixed